GoogleForCreators/web-stories-wp
# Document Panel: Update permalink panel

## Context

Update permalink panel to match [new designs](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=1533%3A260040).

**Designs**
<img src="https://user-images.githubusercontent.com/22185279/111372084-cf211900-865f-11eb-80fb-cd51adb202f8.png" width="850px" />

## Summary

Update permalink panel to match new designs

## Relevant Technical Choices

- input only changes the saved slug on blur. If we want it to submit on each keystroke let me know!

## To-do

none

## User-facing changes

|Designs|Before|After|
|--|--|--|
|![image](https://user-images.githubusercontent.com/22185279/111372395-33dc7380-8660-11eb-84a9-e5b6bdfa1c94.png)|![image](https://user-images.githubusercontent.com/22185279/111372549-62f2e500-8660-11eb-9576-6ce6015ac5a0.png)|![new-permalink](https://user-images.githubusercontent.com/22185279/111374014-3c35ae00-8662-11eb-8236-0e72731b4d43.gif)|


## Testing Instructions

1. Open editor
2. Go to document panel
3. Play with permalink. Should be keyboard accessible.

### QA







1. Open editor
2. Go to document panel
3. Play with permalink. Should be keyboard accessible. User should be able to update their permalink when blurring the input.

### UAT






## Reviews

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist




---



Fixes #6518
